src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -54,10 +54,16 @@ _Health check_ type transactions, while important for checking the stability of
 
 ## Dynamic Sampling & SDK Sampling
 
-Dynamic sampling and your SDK sampling configuration are largely independent of each other. This helps if you're building native applications with many active releases at a given time.
+<Note>
 
-Our dynamic sampling works best when you send us as much data as you can (ideally 100%). As you purchase more transaction volume, however, your per-transaction pricing goes down accordingly.
+When we talk about SDK sample rates in this section, we're only referring to your [transaction sample rate](/platform-redirect/?next=/configuration/sampling/#sampling-transaction-events) (or `tracesSampleRate`), not your [error sample rate](/platform-redirect/?next=/configuration/sampling/#sampling-error-events).
 
-## Next Step
+</Note>
 
-Check out our [Getting Started](/product/data-management-settings/dynamic-sampling/getting-started/) documentation.
+Dynamic sampling and your SDK sampling configuration act independently of each other, but our dynamic sampling works best when you send us as much data as you can. The more complete a picture of your application we have, the more accurately we can monitor your application health, detect problems faster, surface issues, and make better decisions about data retention on your behalf.
+
+If it’s feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume, we’ll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly. Learn more in [Why Dynamic Sampling](/product/data-management-settings/dynamic-sampling/why-dynamic-sampling/)
```

Comment:
```suggestion
If it’s feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume, we’ll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly. Learn more in [Benefits of Dynamic Sampling](/product/data-management-settings/dynamic-sampling/why-dynamic-sampling/).
```